config/features.yml

```diff
@@ -829,6 +829,10 @@ features:
     actor_type: user
     description: Enables notifications to be sent for new copay statements
     enable_in_development: true
+  mhv_account_creation_api:
```

Comment:
we should name this something more descriptive like `mhv_account_creation_after_login`